File: kotlin/PlanesAndroid/app/src/test/java/TypeTest.java
import com.google.common.truth.Truth;
import com.planes.common.Type;

import org.junit.Test;
import static com.google.common.truth.Truth.assertThat;

public class TypeTest {
    @Test
    public void Type_Values() {
        assertThat(Type.values()[0] == Type.Miss).isTrue();
        assertThat(Type.values()[1] == Type.Hit).isTrue();
        assertThat(Type.values()[2] == Type.Dead).isTrue();
    }

}

File: kotlin/PlanesAndroid/app/src/test/java/TypeTest.kt
import com.google.common.truth.Truth
import com.planes.single_player_engine.Type
import org.junit.Test

class TypeTest {
    @Test
    fun Type_Values() {
        Truth.assertThat(Type.values()[0] === Type.Miss).isTrue()
        Truth.assertThat(Type.values()[1] === Type.Hit).isTrue()
        Truth.assertThat(Type.values()[2] === Type.Dead).isTrue()
    }
}